apps/internal/actions.py

```
@@ -0,0 +1,30 @@
+
+    workspace = Workspace.objects.get(fyle_org_id=org_id)
+    workspace_id = workspace.id
+    intacct_credentials = SageIntacctCredential.objects.get(workspace_id=workspace.id)
+
+    return SageIntacctConnector(intacct_credentials, workspace_id)
+
+
+def get_accounting_fields(query_params: Dict):
+    intacct_connection = get_intacct_connection(query_params)
+    resource_type = query_params.get('resource_type')
+    internal_id = query_params.get('internal_id')
```

Ruff (0.7.0)

Local variable `internal_id` is assigned to but never used

Remove assignment to unused variable `internal_id`

(F841)